api/src/main/java/org/apache/iceberg/PartitionKey.java

```diff
@@ -28,11 +28,10 @@
 
 /**
  * A struct of partition values.
- * <p>
+ * <br>
```

Comment:
These should be `<p>`. Could you revert this change?

Reply:
Yeah. I see, I thought it's HTML tag.